pyglider/seaexplorer.py

```diff
@@ -141,6 +141,26 @@ def raw_to_rawnc(indir, outdir, deploymentyaml, incremental=True, min_samples_in
     _log.info('All raw files converted to nc')
     return True
 
+
+def drop_rogue_1970(ds):
+    """
+    If dates greater than 1971, 1, 1 are observed, drop any dates before 1971-01-01, from the datset and return it.
+    This function removes 1970 timestamps caused by a SeaExplorer rebooting during a mission.
+    If all dates are < 1971-01-01, no action is taken
+
+    Parameters:
+        ds: xarray.DataSet
+            dataset to check for pre-1971 dates
+    Returns:
+        ds: xarray.DataSet
+    """
+    dt_1971 = pd.to_datetime(datetime.datetime(1971, 1, 1))
+    # If all dates before or after 1971-01-01, return the dataset
+    if (ds.time > dt_1971).all() or (ds.time < dt_1971).all():
```

Comment:
Is dropping them safe or should we give them better timestamps?   I think it's always a couple of bad times and the little bit of data is safe to drop (and usually not in the ocean), but just checking

Reply:
I just checked with the glider we had that rebooted. Around 0.015 % of the datapoints have 1970 timestamps, and the majority of those have no associated sensor data. I think we're safe to drop the lines. This also avoids the difficulty of interpolating timestamps.